crates/ruff/src/lib.rs

```diff
@@ -29,7 +29,7 @@ mod panic;
 mod printer;
 pub mod resolve;
 mod stdin;
-mod version;
+pub mod version;
```

Comment:
I made this change as I couldn't otherwise figure out how to access functions from the `version` module in `crates/ruff/tests/version.rs`. But this might be a Rust newbie thing 🙃. Lmk if there's a better solution here!